src/core/battle/tests.rs

```diff
@@ -1314,22 +1330,32 @@ fn throw_bomb_push() {
             },
             Event {
                 active_event: event::UseAbility {
-                    id: Id(2),
+                    id: Id(3),
                     pos: PosHex { q: 0, r: 2 },
                     ability: Ability::ExplodePush,
                 }
                 .into(),
-                actor_ids: vec![Id(2)],
+                actor_ids: vec![Id(3)],
                 instant_effects: vec![
                     (
                         Id(1),
                         vec![effect::Knockback {
                             from: PosHex { q: 0, r: 3 },
                             to: PosHex { q: 0, r: 4 },
+                            strength: PushStrength { 0: Weight::Normal },
```

Comment:
push strength is hardcoded in `push_bomb` ability.. ; I notice this test doesn't test the actual moving of the pushed entity, so it's a bit useless to test a "strong" force against normal/heavy/immovable